gulp build runs the "lint" task twice

I noticed that when I run the Docker image, or run `gulp build`, it's running the `lint` task twice (presumably due to task hooks). We could probably save 2-4 seconds by removing the unnecessary linting.

``` bash
$ gulp build

[16:19:04] Using gulpfile ~/dev/github/mozilla/testpilot/gulpfile.js
[16:19:04] Starting 'build'...
[16:19:04] Starting 'clean'...
[16:19:04] Finished 'clean' after 26 ms
[16:19:04] Starting 'app-vendor'...
[16:19:12] Finished 'app-vendor' after 7.85 s
[16:19:12] Starting 'lint'...
[16:19:15]
testpilot/frontend/static-src/app/main.js
  32:3  warning  Unexpected console statement  no-console

✖ 1 problem (0 errors, 1 warning)

[16:19:15] Finished 'lint' after 3.07 s
[16:19:15] Starting 'app-main'...
[16:19:18] Finished 'app-main' after 3.17 s
[16:19:18] Starting 'lint'...
[16:19:20]
testpilot/frontend/static-src/app/main.js
  32:3  warning  Unexpected console statement  no-console

✖ 1 problem (0 errors, 1 warning)

[16:19:20] Finished 'lint' after 1.75 s
[16:19:20] Starting 'scripts'...
[16:19:20] Finished 'scripts' after 73 ms
[16:19:20] Starting 'sass-lint'...
[16:19:20] Finished 'sass-lint' after 497 ms
[16:19:20] Starting 'styles'...
[16:19:21] Finished 'styles' after 1.07 s
[16:19:21] Starting 'images'...
[16:19:21] Finished 'images' after 51 ms
[16:19:21] Starting 'locales'...
[16:19:22] Finished 'locales' after 6.58 ms
[16:19:22] Starting 'addon'...
[16:19:22] Finished 'addon' after 5.75 ms
[16:19:22] Starting 'legal'...
[16:19:22] Finished 'legal' after 20 ms
[16:19:22] Finished 'build' after 18 s
```

--- a/gulpfile.js
+++ b/gulpfile.js
@@ -80,7 +80,7 @@ gulp.task('legal', function legalTask() {
              .pipe(gulp.dest('./legal-copy/'));
 });
 
-gulp.task('app-main', shouldLint('js-lint', 'lint'), function appMainTask() {
+gulp.task('app-main', function appMainTask() {
   return commonBrowserify('app.js', browserify({
     entries: [SRC_PATH + 'app/main.js'],
     debug: IS_DEBUG,

--- a/testpilot/frontend/static-src/app/main.js
+++ b/testpilot/frontend/static-src/app/main.js
@@ -29,7 +29,7 @@ m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
 if (typeof(ga) !== 'undefined') {
   ga('create', 'UA-49796218-34', 'auto');
 } else {
-  console.warn(
+  console.warn( // eslint-disable-line no-console
     'You have google analytics blocked. We understand. Take a ' +
     'look at our privacy policy to see how we handle your data.'
   );

